AFQ/tasks/data.py

```diff
@@ -196,6 +197,33 @@ def csd(subses_dict, dwi_affine,
         tracking_params,
         csd_response=None, csd_sh_order=None,
         csd_lambda_=1, csd_tau=0.1):
+    """
+    full path to a nifti file containing
+    parameters for the CSD fit
+
+    Parameters
+    ----------
+    csd_response : tuple or None, optional.
+        The response function to be used by CSD, as a tuple with two elements.
+        The first is the eigen-values as an (3,) ndarray and the second is
+        the signal value for the response function without diffusion-weighting
+        (i.e. S0). If not provided, auto_response will be used to calculate
+        these values.
+        Default: None
+    csd_sh_order : int or None, optional.
+        default: infer the number of parameters from the number of data
+        volumes, but no larger than 8.
+        Default: None
+    csd_lambda_ : float, optional.
+        weight given to the constrained-positivity regularization part of
+        the deconvolution equation. Default: 1
+    csd_tau : float, optional.
+        threshold controlling the amplitude below which the corresponding
+        fODF is assumed to be zero.  Ideally, tau should be set to
+        zero. However, to improve the stability of the algorithm, tau is
+        set to tau*100 % of the mean fODF amplitude (here, 10% by default)
+        (see [1]_). Default: 0.1
```

Comment:
I think that this reference would not properly get resolved unless the bibliographic item is also included in the docstring.